src/radical/pilot/configs/resource_ncar.json

```diff
@@ -21,21 +21,24 @@
         "agent_config"                : "default",
         "agent_scheduler"             : "CONTINUOUS",
         "agent_spawner"               : "POPEN",
-        "agent_launch_method"         : "MPIRUN",
-        "task_launch_method"          : "MPIRUN",
-        "mpi_launch_method"           : "MPIRUN",
+        "launch_methods"              : {
+                                         "order" : ["FORK", "MPIRUN"],
+                                         "FORK"  : {},
+                                         "MPIRUN": {
+                                             "env_prep": [
```

Comment:
propose to have it as `'setup'`, but to have `env_prep` in sync with Task's `env_prep` also works

Reply:
BTW or should it be "in sync" with TaskDescription's attribute `pre_launch`?